src/courtroom.cpp

```diff
@@ -2008,6 +2010,34 @@ void Courtroom::chatmessage_dequeue()
   unpack_chatmessage(chatmessage_queue.dequeue());
 }
 
+void Courtroom::skip_chatmessage_queue()
+{
+  if (ao_app->is_desyncrhonized_logs_enabled()) {
+    chatmessage_queue.clear();
+    return;
+  }
+
+  while (!chatmessage_queue.isEmpty()) {
+    QStringList p_contents = chatmessage_queue.dequeue();
+    for (int n_string = 0; n_string < MS_MAXIMUM; ++n_string) {
+      // Note that we have added stuff that vanilla clients and servers simply
+      // won't send. So now, we have to check if the thing we want even exists
+      // amongst the packet's content. We also have to check if the server even
+      // supports CCCC's IC features, or if it's just japing us. Also, don't
+      // forget! A size 15 message will have indices from 0 to 14.
```

Comment:
i vaguely recall this comment, where did you copypaste from?

Reply:
Wait did I not commit a less copy pasted approach uhh